src/encoded/static/scss/encoded/modules/_search.scss

```diff
@@ -62,6 +62,10 @@ $form-bg: #f3f3f3;
         position: relative;
     }
 
+    @at-root #{&}__property-title {
+        font-weight: 700;
```

Comment:
Not for now, but in the future, you can use "bold" as bold also has a a value of 700 (https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight)